src/main/java/com/ea/restaurant/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,64 @@
+package com.ea.restaurant.service.impl;
+
+import com.ea.restaurant.constants.Status;
+import com.ea.restaurant.entities.User;
+import com.ea.restaurant.exceptions.EntityNotFoundException;
+import com.ea.restaurant.exceptions.UserNotFoundException;
+import com.ea.restaurant.repository.UserRepository;
+import com.ea.restaurant.service.UserService;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserServiceImpl implements UserService {
+
+  @Autowired private UserRepository userRepository;
```

Comment:
Prefer using injected depencies through constructor, so that way they could be `final`

Reply:
Done.